diff -Naur condor-ec2-enhanced-1.0/config/condor-ec2-enhanced.init condor-ec2-enhanced-1.0f/config/condor-ec2-enhanced.init
--- condor-ec2-enhanced-1.0/config/condor-ec2-enhanced.init	2009-07-22 14:13:56.000000000 -0500
+++ condor-ec2-enhanced-1.0f/config/condor-ec2-enhanced.init	2009-07-22 14:15:26.000000000 -0500
@@ -2,7 +2,7 @@
 #
 # condor-ec2-enhanced   This script allows for stopping and starting the condor
 #                       ec2 enhanced daemon caroniad.
-# chkconfig: 2345 91 11
+# chkconfig: - 91 11
 #
 # description: caroniad is the daemon that translates jobs between condor and
 #              Amazon's Web Services for use with condor's EC2 Enhanced
